pangea-sdk/v3/service/authz/api.go

```diff
@@ -0,0 +1,317 @@
+package authz
+
+import (
+	"context"
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v3/internal/request"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v3/pangea"
+)
+
+type Resource struct {
+	Namespace string `json:"namespace"`
+	ID        string `json:"id,omitempty"`
+}
+
+type Subject struct {
+	Namespace string `json:"namespace"`
+	ID        string `json:"id,omitempty"`
+	Action    string `json:"action,omitempty"`
+}
+
+type Tuple struct {
+	Resource Resource `json:"resource"`
+	Relation string   `json:"relation"`
+	Subject  Subject  `json:"subject"`
+}
+
+type TupleCreateRequest struct {
+	pangea.BaseRequest
+
+	Tuples []Tuple `json:"tuples"`
+}
+
+type TupleCreateResult struct {
+}
+
+// @summary Write tuples.
+//
+// @description Write tuples. The request will fail if there is no schema or the tuples do not validate against the schema.
```

Comment:
Write tuples. The request will fail if tuples do not validate against the schema defined resource types.

I'm updating the API description as well